ocf_datapipes/utils/utils.py

```diff
@@ -19,6 +19,42 @@
 logger = logging.getLogger(__name__)
 
 
+def return_sys_indices_which_has_cont_nan(arr: np.ndarray, check_interval: int = 287) -> np.ndarray:
```

Comment:
```suggestion
def return_system_indices_which_has_contiguous_nan(arr: np.ndarray, check_interval: int = 287) -> np.ndarray:
```

Could the names of the functions be more explicit? Not sure if this should be contiguous or continuous, or something else, but just to keep it as explicit as possible